opentracing/standardtracer/standardimpl.go

```diff
@@ -37,7 +37,7 @@ func (s *standardSpan) StartChild(operationName string) opentracing.Span {
 func (s *standardSpan) SetTag(key string, value interface{}) opentracing.Span {
 	s.lock.Lock()
 	defer s.lock.Unlock()
-	s.raw.Tags[key] = fmt.Sprint(value)
+	s.raw.Tags[key] = value
```

Comment:
Removed conversion to string, as it was making the new tests fail. I think conversion should be done by the Recorder, if necessary.